packages/marketplace/src/styles/blocks/featured-app.scss

```diff
@@ -0,0 +1,3 @@
+.featured-app {
+  min-height: 200px;
```

Comment:
I am not sure why this is required... those images should be responsive - can you explain?

Reply:
When the feature app image hasn't loaded yet, the height of feature app card will be 0 and when the image is loaded, the height of featured app card will be about over 200px so the UI is a little bit ugly. Just add min-height as a placeholder for featured app  card.

Reply:
Makes sense, thanks 👍